src/apps/Agreement/Agreement.js

```diff
@@ -1,28 +1,43 @@
 import React, { useCallback, useMemo, useState } from 'react'
+import PropTypes from 'prop-types'
 import {
   Box,
   Button,
+  formatTokenAmount,
   Header,
   IconEdit,
   IconTrash,
+  noop,
   Split,
   useLayout,
 } from '@aragon/ui'
```

Comment:
```suggestion
import {
  Box,
  Button,
  Header,
  IconEdit,
  IconTrash,
  Split,
  formatTokenAmount,
  noop,
  useLayout,
} from '@aragon/ui'
```

Super minor nitpick here: we push the components first and then the utilities, so it would look like this 😄 .